nilearn/conftest.py

```diff
@@ -12,6 +12,7 @@
 # we need to import these fixtures even if not used in this module
 from nilearn.datasets.tests._testing import request_mocker  # noqa: F401
 from nilearn.datasets.tests._testing import temp_nilearn_data_dir  # noqa: F401
+from nilearn.experimental.surface.tests.conftest import *  # noqa: F401, F403
```

Comment:
I would not be opposed to move the fixture in nilearn/conftest.py rather than doing an "import all".
Any reasons why we should not do it?

Reply:
Not a blocker for a merge.

Reply:
Well we wanted to isolate the new surface module and related code while it's still being developed. But I don't see a way to do that for this PR. I don't like this solution either I just didn't want to rewrite the fixtures and this is temporary. I can instead just import the needed fixtures

Reply:
I made the imports explicit and added a note.